frontend/packages/dev-console/locales/ja/devconsole.json

```diff
@@ -627,14 +627,14 @@
   "Select query": "クエリーの選択",
   "Hide PromQL": "PromQL の非表示",
   "Show PromQL": "PromQL の表示",
-  "Dashboard": "ダッシュボード",
+  "Dashboards": "Dashboard",
```

Comment:
Was this intentional?

Reply:
@TheRealJon I don't think so. Seems like a mistake to me. I have updated with the correct translation. 
PTAL, thanks.